frame/support/src/traits/hooks.rs

```diff
@@ -440,110 +437,4 @@ mod tests {
 			ON_IDLE_INVOCATION_ORDER.clear();
 		}
 	}
-
-	#[cfg(feature = "try-runtime")]
-	#[test]
-	#[allow(dead_code)]
-	fn on_runtime_upgrade_tuple() {
```

Comment:
Why did you removed this?

Reply:
It is not needed anymore, because the logic is now trivial.